test/packages/good/kibana/csp_rule_template/good-csp-rule-template-abc-1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "attributes": {
+    "benchmark_rule_id": "1.1.1",
+    "rego_rule_id": "cis_k8s.cis_1_1_1",
+    "name": "Ensure that the API server pod specification file permissions are set to 644 or more restrictive (Automated)",
+    "description": "'Disable anonymous requests to the API server",
+    "rationale": "When enabled, requests that are not rejected by other configured authentication methods\nare treated as anonymous requests. These requests are then served by the API server. You\nshould rely on authentication to authorize access and disallow anonymous requests.\nIf you are using RBAC authorization, it is generally considered reasonable to allow\nanonymous access to the API Server for health checks and discovery purposes, and hence\nthis recommendation is not scored. However, you should consider whether anonymous\ndiscovery is an acceptable risk for your purposes.",
+    "impact": "Anonymous requests will be rejected.",
+    "default_value": "By default, anonymous access is enabled.",
+    "remediation": "Edit the API server pod specification file /etc/kubernetes/manifests/kubeapiserver.yaml on the master node and set the below parameter.\n--anonymous-auth=false",
+    "enabled": true,
+    "muted": false,
```

Comment:
@joshdover these fields are here as default values for the concrete rules in Kibana.